src/components/input/Input.scss

```diff
@@ -0,0 +1,4 @@
+.form-control {
+  font-size: 0.8rem !important;
```

Comment:
Ogólnie staraj się nie używać _!important_, w poniższym artykule masz wszystko wyjaśnione: [https://element.how/why-you-should-avoid-using-important-in-css](https://element.how/why-you-should-avoid-using-important-in-css)